index.js

```diff
@@ -139,6 +139,10 @@ function callViaDestination(parameters) {
     if (destination.authTokens && destination.authTokens[0]) {
         headers['Authorization'] = `${destination.authTokens[0].type} ${destination.authTokens[0].value}`;
     }
+    
+    //Adding cloud connector name header if passed from request
+    if(`${scc_name}`!==undefined)
+	headers['SAP-Connectivity-SCC-Location_ID'] = `${scc_name}`;
```

Comment:
could you please surround this with curly braces ("{}")?

Reply:
I have made the changes . Kindly review